doc/api/fs.md

```diff
@@ -4923,14 +4923,15 @@ the file contents.
 [Caveats]: #fs_caveats
 [Common System Errors]: errors.html#errors_common_system_errors
 [FS Constants]: #fs_fs_constants_1
+[File Access Constants]: #fs_file_access_constants
+[File Access Constants]: #fs_file_access_constants
```

Comment:
Doc linting issue: `Do not use definitions with the same identifier`.

Reply:
```suggestion
```

Reply:
done